src/files/XmlFile.spec.ts

```diff
@@ -445,38 +491,14 @@ describe('XmlFile', () => {
         expect(actual).deep.equal(expected);
     });
 
-    describe('findExtendsPosition', () => {
```

Comment:
What happened to these tests? It's important that we can properly identify the exact location of the `extends` attribute.

Reply:
I figured that it wasn't needed now that we have a range coming from the AST.

Reply:
The `findExtends` function itself doesn't exist anymore

Reply:
I'll add back equivalent tests